library/src/main/java/com/novoda/downloadmanager/lib/DownloadInfo.java

```diff
@@ -193,53 +80,46 @@ private Long getLong(String column) {
      */
     public static final String EXTRA_IS_WIFI_REQUIRED = "isWifiRequired";
 
-    public long mId;
-    public String mUri;
-    /**
-     * Add to check if scannable i.e. we don't want internal files to be scanned
-     */
-    public boolean mScannable;
-    public boolean mNoIntegrity;
-    public String mHint;
-    public String mFileName;
-    public String mMimeType;
-    public int mDestination;
-    public int mVisibility;
```

Comment:
Visibility is set on the batch level now: https://github.com/novoda/download-manager/blob/master/library/src/main/java/com/novoda/downloadmanager/lib/BatchInfo.java#L7-L8